arduino/netsender/NetSender.cpp

```diff
@@ -25,20 +25,43 @@
 #include <limits.h>
 #include <string.h>
 #include <ctype.h>
+
+#define ESP32
+
+#ifdef ESP8266
 #include <ESP8266WiFi.h>
 #include <ESP8266HTTPClient.h>
 #include <WiFiClient.h>
 #include <EEPROM.h>
-//#include "nonarduino.h"
+#endif
```

Comment:
Consider using ```#elif``` instead

Reply:
I prefer separate #ifdef for each hardware type. Personal preference, I suppose.